docker/transformers-pytorch-deepspeed-latest-gpu/Dockerfile

```diff
@@ -15,6 +15,8 @@ RUN python3 -m pip install --no-cache-dir --upgrade pip
 ARG REF=main
 RUN git clone https://github.com/huggingface/transformers && cd transformers && git checkout $REF
 
+RUN python3 -m pip uninstall -y torch torchvision torchaudio
```

Comment:
just to make the installation in the following lines work more smoothly without the existing torch stuff